src/components/Global/DirectSendQR/index.tsx

```
@@ -1,40 +1,103 @@
-        } else if (isAddress(data)) {
-            // hardcode PNT because this is only used for pinta wallet for now
-            redirectUrl = `${baseUrl}/${data}@polygon/PNT`
-        } else if (validateEnsName(data)) {
-            const resolvedAddress = await resolveFromEnsName(data.toLowerCase())
-            if (!!resolvedAddress) {
-                redirectUrl = `${baseUrl}/${resolvedAddress}@polygon/PNT`
-            }
+        data = data.toLowerCase()
+        switch (recognizeQr(data)) {
+            case EQrType.PEANUT_URL:
+                let path = data
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)